src/cmd-build-tag

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python3 -u
+# Allows users to operate on the tags in `builds.json`
+
+import argparse
+import json
+import os
+import re
+import sys
+
+sys.path.insert(0, '/usr/lib/coreos-assembler')
+from cmdlib import fatal, rfc3339_time, write_json
+
+
+supported_actions = ["create", "delete", "list-builds", "list-tags", "update"]
+
+# Parse args and dispatch
+parser = argparse.ArgumentParser()
+parser.add_argument("--force", action="store_true",
+                    help="Force creating/updating of tag")
+parser.add_argument("action", choices=supported_actions, nargs=1,
+                    help=f"Action to perform")
+parser.add_argument("tag", nargs="?",
+                    help="Tag to operate on")
+parser.add_argument("build", nargs="?",
+                    help="Build to associate tag with")
+args = parser.parse_args()
```

Comment:
Minor/optional: so one way to approach these subcommands is to create subparsers and then assign a function handler for each one. Then you can have truly different arguments/switches per subcommand. See e.g. https://github.com/jlebon/osp-utils/blob/e84a7b97e1cb5bdae5a14c83c6abc4e39ab40ef6/main.py#L24. That should make the code cleaner than using the `if args.action[0] == "my-cmd":` approach.

Reply:
I figured there was a cleaner way to do this than a massive `if` statement.  Thanks for the pointer, I'll dig in.